Doc/library/importlib.metadata.rst

```diff
@@ -187,6 +189,17 @@ function::
     ["pytest (>=3.0.0) ; extra == 'test'", "pytest-cov ; extra == 'test'"]
 
 
+Package distributions
+---------------------
+
+A convience method to resolve the distribution or
+distributions (in the case of a namespace package) for top-level
+Python packages or modules::
+
+    >>> packages_distributions()
```

Comment:
It will be good to document this as newly added in Python 3.10

Reply:
Addressed in 690775ab88.